widget/embedded/src/components/Quote/Quote.tsx

```diff
@@ -279,7 +297,27 @@ export function Quote(props: QuoteProps) {
         listItem={type === 'list-item'}
         basic={type === 'basic'}>
         <div className={summaryStyles()}>
-          <QuoteCostDetails steps={numberOfSteps} quote={quote} />
+          <div id="portal-root" className={summaryHeaderStyles()}>
+            <QuoteCostDetails steps={numberOfSteps} quote={quote} />
+            {!tagHidden &&
+              (tags && tags.length > 2 ? (
+                <Select
+                  container={container}
+                  value={
+                    (selectedTag ?? {
+                      label: i18n.t('Selected'),
+                      value: 'selected',
+                    }) as optionProps
+                  }
+                  options={tags as optionProps[]}
+                  handleItemClick={selectTag}
+                />
```

Comment:
The layout shifts when you open or close the Select component.

Reply:
fixed